### PR TITLE
Keyboard Focus fix to remain on show more button 

### PR DESCRIPTION
**Pull request by Harikarall**
Added a function so that keyboard focus does not move to the title of the product catalog page and narrator announces show more button contents without announcing product catalog 

Testing Details
1.Verified and checked the changes to reflect in the UI locally
2.Verified all unit tests snapshots have passed locally
3.Verified to check if there are no additional issues due to the Change by running Accessibility insights locally.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,50 +28,51 @@
...
-    "tslint": "^5.11.0",
-    "tslint-microsoft-contrib": "^6.0.0",
+    "ts-jest": "^26.5.6",
+    "ts-loader": "^5.4.5",
+    "tslint": "^5.20.1",
+    "tslint-microsoft-contrib": "^6.2.0",
     "tslint-react": "^3.6.0",
-    "typescript": "^3.2.2",
-    "webpack": "^4.27.1"
+    "typescript": "^3.9.10",
```
Comment (PeterYangIO) [inline package.json:74]: TypeScript doesn't follow semantic versioning, so even though npm updated this to the latest minor version, you can really update it all the way to 4.9 if you want.

File: src/PersonaWithCard.test.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { buildProfile } from "./supporting/Profile";
 import { PersonaShowMode, IPersonaProfile } from "./Types";
 import { GraphService } from "./GraphService";
 import { waitNextTick } from "./supporting/Promise";
-import { PersonaSize, HoverCard } from "@fluentui/react";
+import { PersonaSize, HoverCard} from "@fluentui/react";
```
Comment (castrodd) [inline src/PersonaWithCard.test.tsx:7]: Nit: add back whitespace that was deleted.
Also, super-nit: can you put the imports in alphabetical order here (i.e. HoverCard, then PersonaSize)?

**Review by PeterYangIO — APPROVED** [review]

File: src/PersonaWithCard.test.tsx [hunk 3]
```diff
@@ -89,9 +89,10 @@ test("opens link in Compact card", async () => {
 test("renders Expanded card with progress", async () => {
     GraphService.resolveProfile = async () => Promise.resolve(buildProfile(1));
     GraphService.getManager = async () => Promise.resolve(buildProfile(2));
-
-    const openedCard = Enzyme.shallow(<PersonaWithCard id="userId1" showMode={PersonaShowMode.NameOnly} />);
-    openedCard.find(HoverCard).props().onCardVisible();
+    
+    const openedCard =  Enzyme.shallow(<PersonaWithCard id="userId1" showMode={PersonaShowMode.NameOnly} />);
```
Comment (castrodd) [inline src/PersonaWithCard.test.tsx:93]: Nit: spacing is off around the `=` operator.

File: src/PersonaWithCard.test.tsx (hunk 2 above)
Comment (PeterYangIO) [reply src/PersonaWithCard.test.tsx:7]: Consider adding prettier and the space 1es lint rule to this project, then adding it as a PR build step? https://github.com/microsoft-1es/eslint-config-1es-space

File: src/PersonaWithCard.test.tsx [hunk 4]
```diff
@@ -181,21 +182,24 @@ async function renderOpenedCard(profile?: IPersonaProfile) {
     const extractNum = (s: string) => parseInt(s.match(/\d+/)![0], 10);
     GraphService.resolveProfile = async (userId) => Promise.resolve(profile || buildProfile(extractNum(userId)));
     GraphService.getManager = async (userId) => Promise.resolve(buildProfile(extractNum(userId) + 1));
-
     const wrap = Enzyme.shallow(<PersonaWithCard id="userId1" showMode={PersonaShowMode.NameOnly} />);
-    wrap.find(HoverCard).props().onCardVisible();
-
+    const hoverCardProps = wrap.find(HoverCard).props();
+    if(hoverCardProps && hoverCardProps.onCardVisible){
```
Comment (castrodd) [inline src/PersonaWithCard.test.tsx:187]: Nit: need a space after `if` and before `{`

File: src/PersonaWithCard.test.tsx [hunk 5]
```diff
@@ -181,21 +182,24 @@ async function renderOpenedCard(profile?: IPersonaProfile) {
...
+    }
     await waitNextTick();
 
     return wrap;
 }
 
 
 function renderCompactCard(openedCard: Enzyme.ShallowWrapper) {
-    return Enzyme.shallow(<div>{openedCard.find(HoverCard).props().expandingCardProps.onRenderCompactCard()}</div>);
+    const expandingCardProps =  openedCard.find(HoverCard).props()?.expandingCardProps;
```
Comment (castrodd) [inline src/PersonaWithCard.test.tsx:197]: Nit: too many spaces after `=`

File: src/PersonaWithCard.test.tsx [hunk 6]
```diff
@@ -181,21 +182,24 @@ async function renderOpenedCard(profile?: IPersonaProfile) {
...
+    return Enzyme.shallow(<div>{expandingCardProps?.onRenderCompactCard?.()}</div>)
 }
 
 
 function renderExpandedCard(openedCard: Enzyme.ShallowWrapper) {
-    return Enzyme.shallow(<div>{openedCard.find(HoverCard).props().expandingCardProps.onRenderExpandedCard()}</div>);
+    const expandingCardProps =  openedCard.find(HoverCard).props()?.expandingCardProps;
```
Comment (castrodd) [inline src/PersonaWithCard.test.tsx:203]: More space nits

File: src/PersonaWithCard.test.tsx [hunk 7]
```diff
@@ -181,21 +182,24 @@ async function renderOpenedCard(profile?: IPersonaProfile) {
...
+    return expandingCardProps && expandingCardProps.onRenderExpandedCard ? Enzyme.shallow(<div>{expandingCardProps.onRenderExpandedCard()}</div>): <></>;
```
Comment (castrodd) [inline src/PersonaWithCard.test.tsx:204]: Formatting here is a bit wonky. Try:
```
return expandingCardProps && expandingCardProps.onRenderExpandedCard 
    ? Enzyme.shallow(<div>{ expandingCardProps.onRenderExpandedCard() }</div>)
    : <></>;
```

File: src/contactCard/Summary.tsx [hunk 8]
```diff
@@ -12,18 +12,23 @@ export function renderSummary(
...
-): React.ReactNode {
-    return (
+  ): React.ReactNode {
+    function handleContactDetailsClick() {
+      onContactDetailsClick();
+      const button = document.querySelector('[data-focus="button"]');
+      if (button) {
+        (button as HTMLButtonElement).focus();
+      }
+    }
```
Comment (castrodd) [inline src/contactCard/Summary.tsx:22]: Nit: I would add a space before the `return`

File: src/contactCard/Summary.tsx [hunk 9]
```diff
@@ -12,18 +12,23 @@ export function renderSummary(
...
+      return (
         <ul tabIndex={-1} className="summary">
             {renderContactSummary(profile, onContactDetailsClick)}
             {renderOrgSummary(manager, isManagerLoading, onOrgDetailsClick, onPersonaClick)}
         </ul>
-    );
-}
-
-
+    );
```
Comment (castrodd) [inline src/contactCard/Summary.tsx:28]: I think you are missing a closing `}`

File: src/contactCard/Summary.tsx [hunk 10]
```diff
@@ -12,18 +12,23 @@ export function renderSummary(
...
             {renderOrgSummary(manager, isManagerLoading, onOrgDetailsClick, onPersonaClick)}
         </ul>
-    );
-}
-
-
+    );  
+  
 function renderContactSummary(profile: IPersonaProfile, onContactDetailsClick: () => void): React.ReactNode {
-    return (
```
Comment (castrodd) [inline src/contactCard/Summary.tsx:26]: Nit: indentation shouldn't be deleted

File: src/contactCard/Summary.tsx [hunk 11]
```diff
@@ -45,7 +50,7 @@ function renderContactSummary(profile: IPersonaProfile, onContactDetailsClick: (
                 <span>{profile.officeLocation}</span>
                 <span>&nbsp;{profile.city}</span>
             </div>
-            <ActionButton className="more-details contact-details" onClick={onContactDetailsClick}>
+            <ActionButton className="more-details contact-details" onClick={handleContactDetailsClick} aria-label="Show more" aria-expanded={true} data-focus="button" aria-live="polite">
```
Comment (castrodd) [inline src/contactCard/Summary.tsx:53]: Will you be able to call `handleContactDetailsClick` from this function? Is it in scope?

**Review by castrodd — COMMENTED** [review]
@Harikarall I left some comments and questions for you. Let me know if any of them don't make sense.

**Pull request closed without merging**